cypress/integration/basic/group_membership.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Test that group membership can be edited', () => {
```

Comment:
Update describe() and it() language based on https://github.com/dssit/roles-management/pull/349